examples/shader/shader_prepass.rs

```diff
@@ -242,12 +242,7 @@ fn toggle_prepass_view(
             3 => "motion vectors",
             _ => unreachable!(),
         };
-        let text_color = if *prepass_view == 3 {
-            Color::BLACK
-        } else {
-            Color::WHITE
-        };
-
+        let text_color = Color::WHITE;
```

Comment:
Can you inline this variable please? There's no need for it to be a variable anymore.